tests/modules/og_test/src/Plugin/GroupResolver/UserGroupReference.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\og_test\Plugin\GroupResolver;
+
+/**
+ * @file
+ * Contains \Drupal\og_test\Plugin\OgResolver\UserGroupReference.
+ */
+
+use Drupal\og\OgContextBase;
+
+/**
+ * Get the group from a field reference attached to the current user.
+ *
+ * @GroupResolver(
```

Comment:
=> `@OgGroupResolver`? (Just because there could be other `group` of different things - happened already in D7 even with the vertical tabs module or something similar)